dev/routing/users.js

```diff
@@ -39,11 +40,21 @@ module.exports = function (server) {
       },
       handler: function (request, reply) {
         let user = request.payload;
+        let serverUrl = request.headers.referer;
```

Comment:
I am not sure that it is correct way to get host name
https://ru.wikipedia.org/wiki/HTTP_referer